microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md

```diff
@@ -277,6 +277,9 @@ Example:
 
 6. After the policy is created, select **Close**.
 
+> [!WARNING]
+> We have a known issue with the applicability of Attack Surface Reduction on Server OS versions which is marked as compliant without any actual enforcement. There is currently no ETA for when this will be fixed.
```

Comment:
```suggestion
> There is a known issue with the applicability of Attack Surface Reduction on Server OS versions which is marked as compliant without any actual enforcement. Currently, there is no ETA for when this will be fixed.
```